edk2toollib/uefi/edk2/path_utilities_test.py

```diff
@@ -0,0 +1,575 @@
+# @file path_utilities_test.py
+# Contains unit test routines for the path_utilities class.
+#
+#
+# Copyright (c) Microsoft Corporation
+#
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+##
+
+import unittest
+import os
+import sys
+import tempfile
+import shutil
+from edk2toollib.uefi.edk2.path_utilities import Edk2Path
+
+
+class PathUtilitiesTest(unittest.TestCase):
+
+    def MakeFile(self, parentpath, filename):
```

Comment:
MakeFile is a little generic. Maybe a little mores specific or a docstring?

Reply:
I'll make private and add docstring.